config/opal_get_version.m4

```diff
@@ -91,7 +91,7 @@ m4_define([OPAL_GET_VERSION],[
                     $2_REPO_REV=`git describe --tags --always`
                 fi
             else
-                $2_REPO_REV="date`date '+%Y-%m-%d'`"
+                $2_REPO_REV=`$srcdir/getdate.sh`
```

Comment:
Was it intentional to change the `$2_REPO_REV` date format? Who is using this? could this break scripts?

Reply:
Good catch.  Yes, this change in format will break things.

Fix inbound...

Reply:
I dislike the added complexity of the `--ymd`
My fix would have been:

```diff
config/getdate.sh
-date --date="@${SOURCE_DATE_EPOCH:-$(date +%s)}"
+date --date="@${SOURCE_DATE_EPOCH:-$(date +%s)}" "$@"


config/opal_get_version.m4
-  $2_REPO_REV=`$srcdir/getdate.sh`
+  $2_REPO_REV=date`$srcdir/getdate.sh '+%Y-%m-%d'`
```

In the end, it is you guys maintaining it, so you decide which variant you like.

Reply:
Sure, I have no real opinion here.  I'll make the change you suggested.

Reply:
I pushed a small fix to re-add the `date` prefix string to keep the previous format. Looks good to me now.